controllers/doctor.controller.js

```diff
@@ -19,4 +19,18 @@ exports.create = async (req, res) => {
     console.log(err);
     return res.status(500).json({ error: "Something went wrong" });
   }
-};
+}
+
+async function get(req, res) {
```

Comment:
You can use arrow functions:

```
const get = async (req, res) ...
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions
https://medium.com/swlh/javascript-arrow-functions-vs-regular-functions-5ec4a9076796